composer/models/resnet/README.md

```diff
@@ -37,6 +37,9 @@ ResNet family members are defined by their number of layers. Parameter count, ac
 | ResNet-101           | 44.5M           | 78.1%        | 5.50 hrs                 |
 | ResNet-152           | 60.2M           | TBA          | TBA                      |
 
+
+> ❗ **Note**: Please see the :doc:`/model_cards/cifar_resnet` model card for the differences between CIFAR and ImageNet ResNets.
```

Comment:
ah, this won't work since this is a README file -- you'll have to hard link the URL

Reply:
What's the proper url? Is it https://docs.mosaicml.com/en/v0.5.0/model_cards/cifar_resnet.html, or is there a way to make it version-independent (e.g. change v0.5.0 to something else)?

Reply:
`https://docs.mosaicml.com/en/stable/model_cards/cifar_resnet.html`